atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -228,4 +228,14 @@
      * Informs the transaction that a particular table has been written to.
      */
     void useTable(TableReference tableRef, ConstraintCheckable table);
+
+    /**
+     * Disables read-write conflict checking for this table for the duration of this transaction.
```

Comment:
nit: `for this transaction`, or even `for this transaction only` (other tx running concurrently will still do the conflict checks)

Reply:
Done